material-overrides/assets/stylesheets/kluster.css

```diff
@@ -988,12 +1005,62 @@ input.md-search__input[placeholder='Search']::placeholder {
   color: var(--md-footer-fg-color);
 }
 
+/* llms-full button styling */
+.md-footer__buttons {
+  display: flex;
+  gap: 1em;
+  align-items: center;
+  z-index: 3;
+  padding: 0.4rem;
+}
+
+.llms-copy-button {
+  min-width: max-content;
+  display: flex;
+  gap: 1em;
+  align-items: center;
+  padding: 0.4rem;
+  cursor: pointer;
+}
+
+.llms-copy-button svg,
+.llms-copy-button:focus svg {
+  width: 1.25em;
+  height: 1.25em;
+  fill: var(--md-footer-fg-color);
+  display: inline-block;
+  vertical-align: middle;
+  transition: fill 0.3s ease;
+  margin-right: 0.5em;
+}
+
+.llms-copy-button:hover svg {
+  fill: var(--light);
+}
+
+.framer-Bja7l.framer-1v7ikbi.framer-v-1v7ikbi.framer-aotcnd.llms-copy-button {
+  display: flex;
+  align-items: center;
+}
+
 /* Mobile Footer styling */
 @media screen and (max-width: 767px) {
   .md-footer-meta__inner {
     flex-direction: column;
   }
 
+  .md-footer-meta__inner.md-grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-items: center;
+    padding: 0.5rem 0.5rem; 
+  }
```

Comment:
These are targeting the same element. Since the first one is using flex, we can just get rid of it